src/nls/root/strings.js

```diff
@@ -441,6 +446,34 @@ define(function (require, exports, module) {
             122: "F11",
             123: "F12"
         },
+        LIBRARIES: {
+            CREATE_LIBRARY: "Create New Library",
+            RENAME_LIBRARY: "Rename \"%s\"",
+            DELETE_LIBRARY: "Delete \"%s\"",
+            DELETE_LIBRARY_SHARED_CONFIRM: "Your colaborators will no longer have access to this library if you delete it. Are you sure you want to delete \"%s\"?",
+            LEAVE_LIBRARY: "Leave \"%s\"",
+            LEAVE_LIBRARY_CONFIRM: "Are you sure you want to leave \"%s\"?",
+            DELETE_ASSET: "Delete \"%s\"",
+            DELETE_ASSET_CONFIRM: "Are you sure you want to delete \"%s\"?",
+            GRAPHICS: "Graphics",
+            COLORS: "Colors",
+            COLOR_THEMES: "Color Themes",
+            CHAR_STYLES: "Text Styles",
+            LAYER_STYLES: "Layer Styles",
+            BRUSHES: "Brushes",
+            BRUSHES_UNSUPPORTED_1: "You have 1 brush that is not supported.",
+            BRUSHES_UNSUPPORTED_N: "You have %s brushes that are not supported.",
```

Comment:
Might want to change the wording on this a little because it implies that just the particular brushes in your library are not supported. Maybe instead something like "You have %s brushes. Brushes are not supported in Design Space."